lib/src/Uncertainty/Bayesian/Gibbs.cxx

```diff
@@ -40,9 +41,11 @@ Gibbs::Gibbs()
 
 
 /* Parameters constructor */
-Gibbs::Gibbs(const MetropolisHastingsCollection & samplers)
+Gibbs::Gibbs(const MetropolisHastingsCollection & samplers, const Bool isOrderRandom)
```

Comment:
As discussed during the last technical meeting, other sampling methods could be used, e.g. the one used [here](https://github.com/openturns/openturns/blob/3addd4eb174342576e2141a931354548a167942e/lib/src/Uncertainty/Process/GaussianProcess.cxx#L270). In this view, an enum would be more appropriate here. You can see how it is done in the `GaussianProcess` class.

Reply:
Done.